### PR TITLE
Initial support for spanish

### PR DESCRIPTION
**Pull request by yabirgb**
Added first traslations for spanish. Used spanish from Spain (some words are used different in South and Central America but in this case I didn't detect any of this words, at least that I know)

### CODE REVIEW

File: public/locales/es/common.json [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    "depositValue": "Cantidad depositada",
+    "earned": "Ganancias",
+    "marketCap": "Capitalización de mercado",
+    "max": "Max",
+    "pending": "Pendiente",
+    "picklePerBlock": "{{ amount }} PICKLEs son creados en cada bloque.",
+    "picklePerSecond": "{{ amount }} PICKLEs son creados cada segundo.",
+    "poolBalance": "Cantidad acumulada",
+    "poolSize": "Valor acumulado",
+    "poolSizeTooltip": "Total ETH/PICKLE pool value on Uniswap.",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:18]: @yabirgb this is in English, my friend
```suggestion
    "poolSizeTooltip": "Valor total de la ETH/PICKLE pool en Uniswap.",
```

File: public/locales/es/common.json [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    "ratio": "Ratio",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:19]: @yabirgb dunno if this is better. I guess `ratio` could be considered a term of art.
```suggestion
    "ratio": "Relación",
```

File: public/locales/es/common.json [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    "valueStaked": "Cantidad invertida",
+    "walletBalance": "Saldo en la cartera",
+    "yourApy": "Tu APY",
+    "yourBalance": "Tu saldo"
+  },
+  "connection": {
+    "blockNumber": "Este es el número de bloque actual. Esta página se actualiza con cada nuevo bloque.",
+    "changeNetwork": "Cambiar de red",
+    "coinbase": "Coinbase",
+    "connect": "Connect Wallet",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:34]:
```suggestion
    "connect": "Conecta tu Cartera",
```

File: public/locales/es/common.json [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+      "arbitrum": "Arbitrum"
+    },
+    "switchPrompt": "Por favor cambie a la red {{ network }}",
+    "walletConnect": "Wallet Connect"
+  },
+  "dill": {
+    "approveAndCreateLock": "Permitir y crear Lock",
+    "apy": "APY para propietarios de DILL",
+    "averageLockDuration": "El tiempo de congelación medio es actualemente <1>{{ years }}</1> años (basado en {{ dill }} DILL y {{ pickle }} PICKLEs congelados).",
+    "boostCalculator": "PICKLE Boost Calculator",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:53]:
```suggestion
    "boostCalculator": "Calculadora del factor de incentivación de PICKLE",
```

File: public/locales/es/common.json [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+    "dill": "DILL",
+    "dillRequired": "DILL necesario para un incentivo máximo",
+    "expiredSince": "Finalizado (desde {{ date }})",
+    "extending": "Extendiendo...",
+    "extendLockBy": "Extender congelación por",
+    "extendLockTime": "Extender el periodo de congelación",
+    "historic": "Histórico de la distribución de PICKLE",
+    "increaseLockAmount": "Aumentar la cantidad congelada",
+    "lastDistribution": "Distribuidos la semana pasada",
+    "lockedUntil": "Locked (until {{ date }})",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:71]:
```suggestion
    "lockedUntil": "Congelados (hasta el {{ date }})",
```

File: public/locales/es/common.json [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+    "weeklyPickleAmount": "PICKLEs distribuidos semanalmente",
+    "weeklyRevenue": "Ganancias semanales del proyecto",
+    "yourDill": "Tu DILL",
+    "yourShare": "{{ percentage }} del DILL total",
+    "youWillReceive": "Recivirás"
+  },
+  "farms": {
+    "about": "Las granjas te permiten ganar PICKLEs invirtiendo tokens.",
+    "addLiquidity": "Añadir liquidez",
+    "apy": "Coloca el raton sobre la APY para ver de donde provienen las ganancias.",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:102]:
```suggestion
    "apy": "Coloca el ratón sobre la APY para ver de donde provienen las ganancias.",
```

File: public/locales/es/common.json [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+    "picklePower": "Pickle Power",
+    "polygon": {
+      "description": "Las Granjas te permiten obtener incentivos en PICKLE <1/> y MATIC <3/> invirtiendo tokens. (Nota: Los incentivos en MATIC acaban el 23 de Agosto)",
+      "rewards": "Los incentivos son recogidos automáticamente al invertir y retirar la inversión."
+    },
+    "poweredBy": "Gracias a",
+    "showInactive": "Mostrar Granjas inactivas",
+    "showMyJars": "Mostrar mis frascos",
+    "stake": "Invertir",
+    "stakeUnstaked": "Stake Unstaked {{ amount }} Tokens in Farm",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:154]:
```suggestion
    "stakeUnstaked": "Invertir {{ amount }} de tokens no invertidas en la Granja",
```

File: public/locales/es/common.json [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+    "earnedTokens": "Tokens ganados",
+    "earnings": "Ganancias",
+    "emissionRate": "Ratio de emisión: {{ count }} pickle por bloque",
+    "farm": "Granja",
+    "farms": "Granjas",
+    "jar": "Frasco",
+    "jars": "Frascos",
+    "jarValueLocked": "Valor en Frascos",
+    "lastUpdated": "Última actualización",
+    "lifetimeValue": "Lifetime USD Value",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:210]:
```suggestion
    "lifetimeValue": "Valor en USD de por vida",
```

File: public/locales/es/common.json [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+    "month": "Mes",
+    "monthly": "{{ percent }}% monthly",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:212]:
```suggestion
    "monthly": "{{ percent }}% mensual",
```

File: public/locales/es/common.json [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+    "twitter": "Twitter",
+    "vote": "Votar"
+  },
+  "oldFarms": {
+    "back": "a nuevas granjas",
+    "disclaimer": "Estas granjas han parado de ganar $PICKLEs el Jue 22 de Abril de 2021 (12 AM GMT) <br /> Para ganar $PICKLEs, migra cualquier token aquí to a las nuevas granjas <br /> Despliega en cualquier granja donde tengas tokens y haz click en <strong>Migrar</strong>"
+  },
+  "time": {
+    "day": "1 día",
+    "day_plural": "{{ count }} dias",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:253]:
```suggestion
    "day_plural": "{{ count }} días",
```

File: public/locales/es/common.json [hunk 11]
```diff
@@ -0,0 +1,272 @@
...
+    "month": "1 mes",
+    "month_plural": "{{ count }} meses",
+    "week": "1 semana",
+    "week_plural": "{{ count }} semanas",
+    "year": "1 año",
+    "year_plural": "{{ count }} años"
+  },
+  "zap": {
+    "deposit": "Depositar Token",
+    "depositing": "Depositando en la granaja...",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:263]:
```suggestion
    "depositing": "Depositando en la Granja...",
```

File: public/locales/es/common.json [hunk 12]
```diff
@@ -0,0 +1,272 @@
...
+    "description": "Zap ETH o CRV en ETH/yvBOOST SLP y lo deposita en la <1>Pickle Farm</1>.",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:264]:
```suggestion
    "description": "Zap ETH o CRV en ETH/yvBOOST SLP y lo deposita en la <1>Granja de Pickle</1>.",
```

File: public/locales/es/common.json [hunk 13]
```diff
@@ -0,0 +1,272 @@
...
+    "notAvailable": "Zapping disponible en la red principal",
+    "zap": "Zap",
+    "zapAndStake": "Zap e invertir",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:267]:
```suggestion
    "zapAndStake": "Zap e Invertir",
```

File: public/locales/es/common.json [hunk 14]
```diff
@@ -0,0 +1,272 @@
...
+    "zapIntoJar": "Zap a un frasco",
```
Comment (mannyreimi88) [inline public/locales/es/common.json:268]:
```suggestion
    "zapIntoJar": "Zap al Frasco",
```

**Review by mannyreimi88 — CHANGES_REQUESTED** [review]
@yabirgb thanks for this work. A few omissions and other suggestions.

File: public/locales/es/common.json (hunk 3 above)
Comment (yabirgb) [reply public/locales/es/common.json:34]: Thanks! I believed it was another wallet name

File: public/locales/es/common.json [hunk 15]
```diff
@@ -0,0 +1,272 @@
...
+    "medium": "Medium",
+    "pickleEth": "PICKLE-ETH",
+    "profit": "Ganancias",
+    "stats": "Estadísticas",
+    "twitter": "Twitter",
+    "vote": "Votar"
+  },
+  "oldFarms": {
+    "back": "a nuevas granjas",
+    "disclaimer": "Estas granjas han parado de ganar $PICKLEs el Jue 22 de Abril de 2021 (12 AM GMT) <br /> Para ganar $PICKLEs, migra cualquier token aquí to a las nuevas granjas <br /> Despliega en cualquier granja donde tengas tokens y haz click en <strong>Migrar</strong>"
```
Comment (janklimo) [inline public/locales/es/common.json:249]: @mannyreimi88 is this redundant?

```suggestion
    "disclaimer": "Estas granjas han parado de ganar $PICKLEs el Jue 22 de Abril de 2021 (12 AM GMT) <br /> Para ganar $PICKLEs, migra cualquier token aquí a las nuevas granjas <br /> Despliega en cualquier granja donde tengas tokens y haz click en <strong>Migrar</strong>"
```

Comment (yabirgb) [reply public/locales/es/common.json:249]: Yes, that to doesn't exists in spanish. Thanks for spotting it

**Comment by janklimo** [comment]
Thank you @yabirgb 💯

**Pull request merged**